Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -916,6 +916,39 @@
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitClown
 
+#Mime Hardsuit
+- type: entity
+  parent: ClothingOuterHardsuitBase
```

Comment:
you can just change the parent from ClothingOuterHardsuitBase to ClothingOuterHardsuitClown to reduce the amount of text, since your spacesuit completely replicates the clown's spacesuit in characteristics...

Someday people will give up Ctrl C + Ctrl V...